tests/suite/zar/s3/minio.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+mkdir -p data/bucket
+minio server --writeportfile="./port" --quiet --address ":0" ./data &
```

Comment:
Is it possible to use 'localhost:0' as the minio address, to avoid system popup warnings about listening to network ports?

Reply:
sure